package.json

```diff
@@ -2,12 +2,16 @@
   "name": "tokybook-dl",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "dist/index.js",
   "scripts": {
-    "start": "node index.js",
+    "start": "node src/index.js",
+    "build": "tsc",
+    "debug": "node --inspect=6666 -r ts-node/register index.js",
     "test": "eslint .",
+    "posttest": "npm run build",
```

Comment:
This and the `prepublishOnly` scripts are things I can also see as being up for decision. I like building after before every commit to encourage clean commits, but I can see the argument against this as the project increases in size. Let me know if we don't like this, and I'll remove this script and make the `prepublishOnly` into something like `npm run test && npm run build`.